drivers/tests/UNITTESTS/SFDP/test_sfdp.cpp

```diff
@@ -0,0 +1,101 @@
+/* Copyright (c) 2020 ARM Limited
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "gtest/gtest.h"
+#include "gmock/gmock.h"
+#include "drivers/internal/SFDP.h"
+
+class TestSFDP : public testing::Test {
+protected:
+    struct mbed::sfdp_smptbl_info smptbl;
+
+    /**
+     * Construct Mbed OS SFDP info.
+     * Normally this is parsed from the flash-chips's memory-mapped
```

Comment:
```suggestion
     * Normally this is parsed from the flash-chips's
```
It's not memory mapped, but rather fetched by an SPI/QSPI/OSPI command. I'll correct it after receiving some review.

Reply:
Done